src/app/new-auth/components/error-banner/error-banner.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input, OnInit} from '@angular/core';
+import { AuthValidationService } from 'app/core/auth-validation.service';
+@Component({
+  selector: 'clark-error-banner',
+  templateUrl: './error-banner.component.html',
+  styleUrls: ['./error-banner.component.scss']
+})
+export class ErrorBannerComponent implements OnInit{
+
+  isError: Boolean = true;
+  @Input() errorMessage: String = 'You have 5 login attempts per hour.';
```

Comment:
In my opinion, we should also let the user know how many attempts they have left.